media/js/exp/firefox/new/desktop/join-modal.js

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var client = window.Mozilla.Client;
+    var joinFirefoxContent = document.querySelector('.join-firefox-content');
+
+    function showFxAModal(e) {
+        e.preventDefault();
+
+        // Open the modal
+        Mzp.Modal.createModal(this, joinFirefoxContent, {
+            title: e.target.textContent,
+            className: 'join-firefox-modal'
+        });
+
+        // Count the click in GA
+        window.dataLayer.push({
```

Comment:
@alexgibson Should I add an experiment specific value to this dataLayer push?

Reply:
THe event should be tied to the URL in GA already, so I don't think there should be anything extra required (although I'm no GA expert)

Reply:
That sounds right. Thanks! I think I've addressed all the feedback now.